src/content/structured/components/cards/guidance.mdx

```diff
@@ -241,7 +241,7 @@ Images should always feature an `alt` attribute. Set the `alt` equal to a descri
 
 ### Content area
 
-If placing several lines of text in the content area, use a “see more” link to make the content expandable.
+If placing several lines of text in the content area, use a 'see more' link to make the content expandable.
```

Comment:
As mentioned above, there are inconsistencies withe 'See more' and 'see more'. I would tend towards the 'see more' so that the text is sentence case.